docs/saas.rst

```diff
@@ -153,6 +153,9 @@ The following is a complete example of using the xOpera SaaS API through ``curl`
     # final login step, sets cookies and automatically completes the /secret request
     cookiecurl "$redirect_url" -d "username=$your_username" -d "password=$your_password" -d credentialId=""
 
+    # xopera-saas requires you to be mindful
```

Comment:
This could be also mentioned somewhere in the documentation above.

Reply:
Done! I've put it in the warnings section at the bottom, because it applies to all kinds of access to xopera-saas.

Reply:
Great!